src/get-started/wordcount-example.md

```diff
@@ -245,12 +302,15 @@ The Minimal WordCount pipeline contains five transforms:
     ```py
     {% github_sample /apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets.py tag:examples_wordcount_minimal_write
     %}```
-    
-Note that the `Write` transform produces a trivial result value of type `PDone`, which in this case is ignored.
+
+Note that the `Write` transform produces a trivial result value of type `PDone`,
+which in this case is ignored.
 
 ### Running the Pipeline
 
-Run the pipeline by calling the `run` method, which sends your pipeline to be executed by the pipeline runner that you specified when you created your pipeline.
+Run the pipeline by calling the `run` method, which sends your pipeline to be
+executed by the pipeline runner that you specified when you created your
```

Comment:
when you created your pipeline -> in your `PipelineOptions`

Reply:
done